scripts/deploy/gke.sh

```diff
@@ -44,20 +52,21 @@ Options:
   --stop-proxy        Stop a proxy container
 
 Environment variables:
-  X1_CLUSTER_NAME                Cluster name, must be unique in the region, default is x1-$USER
+  X1_CLUSTER_NAME                Cluster name, must be unique in the region, default is icl-$USER
   X1_GCP_PROJECT_NAME            GCP project name
   X1_GCP_ZONE                    GCP zone to use, default is us-central1-a
-  ICL_INGRESS_DOMAIN             Domain for ingress, default is test.x1infra.com
-  GOOGLE_APPLICATION_CREDENTIALS Location of a Google Cloud credential JSON file.
-  ICL_GCP_MACHINE_TYPE           Machine type for GKE to use
-  TF_PG_CONN_STR                 If set, PostgreSQL backend will be used to store Terraform state 
-  PGUSER                         PostgreSQL username for Terraform state
-  PGPASSWORD                     PostgreSQL password for Terraform state
+  ICL_INGRESS_DOMAIN              Domain for ingress, default is test.x1infra.com
+  GOOGLE_APPLICATION_CREDENTIALS  Location of a Google Cloud credential JSON file.
+  ICL_GCP_MACHINE_TYPE            Machine type for GKE to use
+  GPU_MODEL                       GPU being requested from GCP (e.g. nvidia-tesla-t4)
+  TF_PG_CONN_STR                  If set, PostgreSQL backend will be used to store Terraform state 
+  PGUSER                          PostgreSQL username for Terraform state
+  PGPASSWORD                      PostgreSQL password for Terraform state
```

Comment:
Here one line is added, but because the formatting changed for other lines it is hard to see what is the actual change. Try to keep your changes local ()
```suggestion
  ICL_INGRESS_DOMAIN             Domain for ingress, default is test.x1infra.com
  GOOGLE_APPLICATION_CREDENTIALS Location of a Google Cloud credential JSON file.
  ICL_GCP_MACHINE_TYPE           Machine type for GKE to use
  GPU_MODEL                      GPU being requested from GCP (e.g. nvidia-tesla-t4)
  TF_PG_CONN_STR                 If set, PostgreSQL backend will be used to store Terraform state 
  PGUSER                         PostgreSQL username for Terraform state
  PGPASSWORD                     PostgreSQL password for Terraform state
```

Reply:
Resolved in commit:
https://github.com/intel-ai/icl/pull/76/commits/0de90d5ba55ef61cead0e0144be856e661c7e49d